index.html

```diff
@@ -3954,6 +3954,17 @@ <h4>equivalentId</h4>
 <code><a>equivalentId</a></code> DID and the <code>id</code> property DID.
                   </p>
                 </div>
+                
+                <div class="issue" title="Observance of equivalentId recommendations">
+                  <p>
+If a resolving party does not retain the values from the <code>id</code> and
+<code><a>equivalentId</a></code> properties and ensure any subsequent
+interactions with any of the values they contain are correctly handled as
```

Comment:
Also, to be clear -- Editor hat off -- equivalentId places a possibly unacceptable burden on RPs. I doubt Digital Bazaar will support any DID Method that uses the feature as it makes the lookup tables for account information very complex and potentially error prone. 

I expect other RP implementers to get the implementation wrong, which will result in security failures that are not the failure of the DID Method, but will most likely be blamed on the DID Method. I don't expect that canonicalId has the same issue.